controllers/exotics.js

```diff
@@ -0,0 +1,106 @@
+const models = require('../models')
+
+const getAllExotics = async (request, response) => {
+  try {
+    const exotics = await models.Exotics.findAll()
+
+    return exotics
+      ? response.send(exotics)
+      : response.sendStatus(404)
+  } catch (error) {
+    return response.status(500).send('Unable to get exotics list, Please try again.')
+  }
+}
+
+const getExoticByName = async (request, response) => {
+  try {
+    const { name } = request.params
+
+    const exotic = await models.Exotics.findOne({
+      include: [{ model: models.Hunters }],
+      where: {
+        name: { [models.Op.like]: `%${name}%` }
+      }
```

Comment:
If you used an OR here you could have supported both name and type the way your docs claim. You should review that module from week 13.